### PR TITLE
Events documentation

### PR DESCRIPTION
**Pull request by prog8**

### CODE REVIEW

File: docs/events/timeline.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title:Timeline/Histogram
+
+All events are time-based. This means they are presented on a histogram (aka timeline). Chart
+contains events grouped by event type. You can rely on colors presented on the chart to quickly
+figure out what are the most common event types at each point in time. Events chart behaves
+exactly in the same way as all other charts in [Sematext Cloud](http://sematext.com/cloud/).
```
Comment (otisg) [inline docs/events/timeline.md]: exactly in the same way => the same
http => https

File: docs/events/timeline.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Events are listed in Events Stream just below the chart. Each event is associated with a single
+Sematext App so they are marked with an icon.
```
Comment (otisg) [inline docs/events/timeline.md]: Each event is associated with a single Sematext App so they are marked with an icon => Each event is associated with a single Sematext App whose icon is displayed alongside the event.

File: docs/events/index.md [hunk 3]
```diff
@@ -17,173 +17,23 @@ filtered.
...
 Beyond events that you want to see as part of your operations
 intelligence think about events that matter to your team or your
-organization in general.  Such "business events" can be shipped to
-Sematext, too.
+organization in general.  All kinds of "business events" can be shipped
+to Sematext, too. [Read](/events/adding#adding-events-through-ui) more about
+adding custom events. You can create an event when your web application encounters
+an outage and when the problem which caused the outage has been fixed.
 
 Besides being shown in the UI events are also exposed via a [REST API](../api)
```
Comment (adnanrahic) [inline docs/events/index.md]: the UI events -> the UI, events

**Comment by adnanrahic** [comment]
Hey @prog8, this is great! Could you create a new PR and request to merge into `develop` instead of the `master` branch?

**Comment by prog8** [comment]
Sorry I did it by accident.

**Pull request closed without merging**